Resources/Textures/Objects/Consumable/Drinks/robustnukie.rsi/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by PuroSlavKing (github) for SS14",
+  "states": [
+    {
+      "name": "icon"
+    },
+	  {
+	    "name": "icon_open"
+  	},
+    {
+      "name": "inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "inhand-left",
+      "directions": 4
+    }
+  ]
+}
```

Comment:
Consistent formatting and newline at the end plz I beg.

Reply:
> Consistent formatting and newline at the end plz I beg.

Added a new line. I used the meta can of cola, it's in exactly the same order :*